[Repo] Upstream sync

DEATHB4DEFEAT opened the pull request:
Woohoo, we are [going to be] up to date with Wizden!

All the issues with this that I know of:
- [x] Maps won't load
- [x] Maps need to be resaved
- ~~Traitor codewords don't work~~ (Can't repro)
- ~~Flawed doesn't get any objective~~ (Can't repro)
- [x] Boxes are all invisible
- ~~Need to update emergency shuttle~~
- [x] Fix the client "ghost rsi" error
- [x] Fix spriteview
- [x] Fix announcements

Cheackraze committed:
Lava Brig Dungeon Theme (#14637)

metalgearsloth committed:
Add more dungeon layouts (#14924)

metalgearsloth committed:
Don't sell non-dead mobs on cargo (#15228)

Scribbles0 committed:
[Antag] Lone Ops (#14647)

* loneops event, prototype stuff, striker shuttle, and nukeops rule changes

* newline

* shuttle attributions

* optimizations and tweaks

* bugfix and mutually exclusive with nukeops

* bugfix but better

* fix nukie planet spawning when defaulting to extended

* remove hypospray protection references

* ghost_roles.yml edit thingy

* remove .orig file

Scribbles0 committed:
[fix] lone ops fix (#15478)

Nimfar11 committed:
[Entity] SWAT gas mask for HoS (#15539)

* SWAT gas mask for HoS

Adds a SWAT gas mask. Replaces the standard Security gas mask in the HoC box with an elite one.

* fix

* commit

* commit2

* commit3

metalgearsloth committed:
Fix cargo FTL (#15557)

metalgearsloth committed:
Expedition loc + timer fixes (#15561)

deltanedas committed:
fix loneop sprite path (#15570)

metalgearsloth committed:
Fix accidentally selling mobs (#15578)

metalgearsloth committed:
Expedition mining spawn adjustments (#15581)

metalgearsloth committed:
Expeditions QOL (#15599)

metalgearsloth committed:
Don't enable ghost roles for salvage spawns (#15598)

metalgearsloth committed:
Add salvage song (#15580)

metalgearsloth committed:
Add salvage shuttle (#15609)

metalgearsloth committed:
More salvage fixes (#15618)

EmoGarbage404 committed:
[entity] Mega Spray Bottle (#15558)

metalgearsloth committed:
Reduce expedition time (#15624)

DogZeroX committed:
Adds shiny crystal for salvage and decoration (#15643)

EmoGarbage404 committed:
Anomaly Locator (#15677)

metalgearsloth committed:
Fix tests (#15682)

Jackal298 committed:
add salvage rig (#15656)

metalgearsloth committed:
Update salvage.ogg (#15688)

EmoGarbage404 committed:
CHIMP handcannon (#15667)

EmoGarbage404 committed:
printed health analyzers and crew monitors no longer have batteries (#15699)

freeman2651 committed:
[tweak] Medkit inventory change (#15636)

* rebalance medkit contents

* and top of the empty space

EmoGarbage404 committed:
Make PowerCellDraw not tick dependent, buff anomaly locator power drain (#15719)

EmoGarbage404 committed:
Gamerule Entities, Take 2 (#15765)

tom-leys committed:
Fix 15710, can now admin ctrl | traitor (#15875)

EmoGarbage404 committed:
Fix loneops (#15874)

EmoGarbage404 committed:
Fix station events that use update loops (#15834)

metalgearsloth committed:
Update submodule to 0.103.0.0 (#15879)

TemporalOroboros committed:
Resolves PowerChargerVisualizer is Obsolete (#13901)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

deltanedas committed:
Uplink store has to be open to receive TC (#14341)

Co-authored-by: deltanedas <@deltanedas:kde.org>

EmoGarbage404 committed:
Convert materials to use PhysicalComposition (#15414)

metalgearsloth committed:
Cargo 1984 (#15579)

deltanedas committed:
add borg legs (#15596)

Co-authored-by: deltanedas <@deltanedas:kde.org>

ElectroJr committed:
Blindness refactor (#15705)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

Tunguso4ka committed:
bonfire and mattress (#15871)

jamessimo committed:
Update AnomGen UI (#14872)

ElectroJr committed:
Generalize Sprite Views (#15881)

metalgearsloth committed:
Update submodule to 0.104.0.0 (#15882)

lzk228 committed:
Added ghost sheet craft (#15281)

VasilisThePikachu committed:
Add looc and emote focus binds (#15588)

ElectroJr committed:
Prevent brains from walking (#15709)

Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

metalgearsloth committed:
Remove deprecated NPC debug buttons (#15824)

metalgearsloth committed:
Fix master (#15883)

metalgearsloth committed:
Refactor thrusters (#15698)

Vaaankas committed:
rmobjective fix (#15850)

Co-authored-by: vanx <vanx#5477>

potato1234x committed:
Salt & Pepper Shakers (#15808)

pofitlo committed:
add new music for nuke (#15035)

ElectroJr committed:
Remove SpriteChange construction action (#15887)

deltanedas committed:
no more clown satchel + duffel trolling (#15831)

Co-authored-by: deltanedas <@deltanedas:kde.org>

ElectroJr committed:
Remove server-side sprite references from botany system (#15886)

Bixkitts committed:
Converted BurnState Visualizer (#15661)

ElectroJr committed:
Remove server-side sprite references from chem master (#15888)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

metalgearsloth committed:
Remove pill tags (#15891)

metalgearsloth committed:
Update submodule to 0.104.1.0 (#15894)

ElectroJr committed:
Remove server-side sprite references from Flash system (#15893)

EmoGarbage404 committed:
Make anomaly scanners show exact output (#15749)

notquitehadouken committed:
Add windowed shuttle door (#15873)

Co-authored-by: notquitehadouken <1isthisameme>

deltanedas committed:
flash uses charges system now (#15898)

Co-authored-by: deltanedas <@deltanedas:kde.org>

vulppine committed:
Adds formatting issue analyzing in Omnisharp (VSCode) (#15809)

Slava0135 committed:
Electrified grille sparks effect (#15178)

* use file namespace

* shorter systems name

* replace SoundSystem with AudioSystem

* refactor update function

* refactor

* refactor 2

* remove setters

* uh oh

* remove getters

* active checks

* refactor 3

* better way

* update state

* have to remove this for now

* move electrified component to shared

* forgot this

* fix airlocks

* add effect

* Revert "move electrified component to shared"

This reverts commit 6457e8fc9c3b674a705a61034831ce6f084e2b01.

* Revert "forgot this"

This reverts commit ed361cee2d5b8b958830ba0af07fcc2627eb7845.

* functioning effects

* use animation by Aleksh

* make effect part of grille

* optimisation?

* remove timing

* file name

* only activate when touched

* refactor electrocution comp too

* make it 1 sec

* formatting

* replace all entity query with enumerator

* queue del

Interrobang01 committed:
Impedrezene now actually works (#14843)

* Impedrezene now actually does stuff

* made impedrezene waaay harder to make

* Replaced genetic damage with poison damage, vision problems, and vomiting.

* Made recipe easier by replacing potassium requirement with oxygen requirement

* lowered vomiting chance

* quartered the potency of the drunkenness effect

ElectroJr committed:
Replace `SpriteStateChange` construction action with `AppearanceChange` (#15914)

DawBla committed:
Fix 0.25sec doafters (#15918)

LudwigVonChesterfield committed:
Fixes Typo In Station Event Documentation. (#15932)

ElectroJr committed:
Make blurry vision darken the screen (#15926)

ElectroJr committed:
Prevent follower recursion (#15921)

EmoGarbage404 committed:
Fix instant power cell drainage (#15927)

ElectroJr committed:
Remove server and shared sprite component (#15917)

metalgearsloth committed:
Update submodule to 0.105.0.0 (#15938)

metalgearsloth committed:
Fix gun dump bug (#15936)

ElectroJr committed:
Set SpriteView sizes in various controls. (#15935)

metalgearsloth committed:
Always write for UninitializedSaveTest (#15937)

AJCM-git committed:
Tweaks to ContainerAmmoProvider (#15945)

metalgearsloth committed:
Remove flare flickering (#15787)

It's soooooo annoying.

Menshin committed:
Cached barotrauma resistance and immunity values instead of computing them each Update() (#15055)

Scribbles0 committed:
[entity] new gas miner prototypes (#15916)

Jackal298 committed:
Changing the side sprites of tactical vests (#15870)

* Tactical vests

* two pixels

metalgearsloth committed:
Update submodule to 0.105.1.0 (#15960)

ElectroJr committed:
Add basic map loader support for entity renaming (#15862)

Pspritechologist commented on Tools/gen_build_info.py:
```diff
@@ -21,7 +21,7 @@
 
 VERSION = os.environ['GITHUB_SHA']
 FORK_ID = "nyanotrasen"
-BUILD_URL = f"https://builds.nyanotrasen.moe/{{FORK_ID}}/builds/{{FORK_VERSION}}/{FILE}"
+BUILD_URL = f"https://cdn.nyanotrasen.moe/builds/{{FORK_VERSION}}/{FILE}"
 MANIFEST_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/manifest"
 MANIFEST_DOWNLOAD_URL = f"https://cdn.nyanotrasen.moe/version/{{FORK_VERSION}}/download"
```
Is this something we want to change? What do they do?

DEATHB4DEFEAT replied:
optional infrastructure we don't have

Pspritechologist replied:
We should probably tweak it just for the sake of not having their infrastructure in our repo. Comment it out, leave or blank, or whatever makes sense.

DawBla committed:
[fix] ammo typo(#15954)

metalgearsloth committed:
Content fixes for timeoffsets (#15934)

metalgearsloth committed:
Update submodule to 0.106.0.0 (#15961)

AJCM-git committed:
Moves buckling and vehicles to shared, some cleanup (#15923)

AJCM-git committed:
Small tweak and cleanup of spells and actions (#15943)

metalgearsloth committed:
Mobmover tweaks (#15942)

deltanedas committed:
rcd refactor (#15172)

Co-authored-by: deltanedas <@deltanedas:kde.org>

lzk228 committed:
Nuke agent is separete role (#15385)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

metalgearsloth committed:
Space cleaner buffs (#15779)

metalgearsloth committed:
Cleanbot tweaks (#15821)

TemporalOroboros committed:
Resolves ExpendableLightVisualizer is Obsolete (#13888)

minism committed:
Refactor SMES component to move logic into PowerSmesSystem. (#15438)

Vordenburg committed:
[Sci] Non-destructive XenoArch Research (#15398)

* Non-destructive XenoArch research

* nerf the price

* Points -> Extract

NOTE: XenoArch is still destructive for Nyanotrasen.

ProPandaBear committed:
Toxin damage tweak (#15524)

metalgearsloth committed:
Fix server crash for name identifiers (#15584)

Bixkitts committed:
Convert ReinforcedWallVisualizer to ECS (#15735)

PJB3005 committed:
Database thread pool use (#14498)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

themias committed:
Fix cutting/pulsing high-sec door wires (#15989)

DawBla committed:
Fake capgun discount (#15995)

ElectroJr committed:
Prevent mutation of non-unique plant seeds (#15702)

brainfood1183 committed:
Omega Cartridges Uplink rounds for CHIMP. (#15755)

Co-authored-by: Nemanja <98561806+EmoGarbage404@users.noreply.github.com>
Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

lzk228 committed:
[Sprite] inhand sprites for bags (#16005)

lzk228 committed:
Crowbar for ERT (#16011)

Menshin committed:
Fixes zipties spawning multiple entities on removal (#15761)

Slava0135 committed:
Round end zombie percentage changes (#15620)

metalgearsloth committed:
Make paused poweredlights enabled (#14270)

Slava0135 committed:
Explosive grenade sound (#15582)

Vordenburg committed:
Fix heater and freezer temperature range and wrong unit (#15904)

NOTE: This merges the fix for the wrong unit but not the changes
dependent on #15802

EmoGarbage404 committed:
Defibrillator (#15922)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

metalgearsloth committed:
Update submodule to 0.107.0.0 (#16027)

metalgearsloth committed:
Defib fixes (#16031)

DjfjdfofdjfjD committed:
[map] Update Inaprovaline conditions (#16040)

EmoGarbage404 committed:
[fix] Fix rotting (#16039)

* Fix rotting

* empty

Interrobang01 committed:
[fix] removing the stray disabler pixel (#16041)

Slava0135 committed:
Bomb suit explosion resistance (#15732)

EmoGarbage404 committed:
Reduce vendor restocking time + some code cleanup (#16025)

Naive817 committed:
adds damp rag (#15553)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

notquitehadouken committed:
Make dwarves rot (#16050)

themias committed:
Stop mimes from screaming or laughing (#16052)

ThunderBear2006 committed:
Ice anomaly (#15925)

Co-authored-by: Nemanja <98561806+EmoGarbage404@users.noreply.github.com>

Shadowtheprotogen546 committed:
The Martinez drink from cyberpunk 2077 (#15421)

* added 'The Martinez' drink made with 1 ice, 1 lemon juice, 2 vodka, and 2 cola
---------

EmoGarbage404 committed:
Light replacer cleanup + examine (#16066)

metalgearsloth committed:
Flare gun 1984 (#15807)

metalgearsloth committed:
Update submodule to 0.107.0.1 (#16084)

lzk228 committed:
Add mime's stamp in satchel (#16044)

Pspritechologist commented on Resources/Server Info/RedialAddresses.txt:
```diff
@@ -1,6 +1,9 @@
-# ss14://nyanotrasen.moe:1212
-# ss14://nyanotrasen.moe:1213
-# ss14://nfn.mooo.com:1213
+# Nyano
+ss14s://io.nyanotrasen.moe
+# Mining Station
+ss14://nfn.mooo.com:1213
+# Delta-V
+ss14://delta-v.org:1212
```
Huh! Neat.

Pspritechologist replied:
Huh! Neat.

SonicHDC committed:
Paintable Glass Airlocks (#16004)

Pspritechologist reviewed (approved):
Mostly reviewing so if you push more in the future, I can compare exactly what it was :P
Take a look at the redial thing.

Also, until more gets pushed, feel free to merge this whenever.

themias committed:
Fix duplicate verbs on janitor trolley (#16087)

lzk228 committed:
Update cat crate (#16074)

Scribbles0 committed:
A few trait fixes (#16062)

Partmedia committed:
Reparent crafted items (#15848)

ElectroJr committed:
Rollerbed / foldable strap fixes. (#16106)

Slava0135 committed:
display item charge on examine (#16116)

Slava0135 committed:
Fix fire extinguisher (#16113)

tom-leys committed:
Split FleshAnomaly into two components #16001 (#16110)

Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

Slava0135 committed:
prevent from prying blast doors (#16077)

Ilushkins33 committed:
Stuttering during blood loss. (#15153)

Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

Vordenburg committed:
Misc sprite changes (#15011)

We'll get a different fax machine sprite later.

metalgearsloth committed:
Set station suit sensors on player spawn (#16111)

themias committed:
Handheld Crew Monitor and Station Map use power (#15983)

PJB3005 committed:
Fix redundant null check (#16131)

Scribbles0 committed:
Reduce lighter fuel consumption on light (#16129)

EmoGarbage404 committed:
fix events starting instantly (#16134)

EmoGarbage404 committed:
Spray Nozzle & Backpack Water Tank (#16133)

EmoGarbage404 committed:
Fix chatty zombies (#16139)

tom-leys committed:
Fix firelock lights for hot / pressurized rooms (#15892)

lzk228 committed:
Increase wet floor holosign lifetime (#16148)

lapatison committed:
Cig pack and matchbox tweaks (#15538)

Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

Samsterious committed:
Enhance paperwork UI (#16006)

lapatison committed:
[Frozen] Soups tweaks (#15110)

Co-authored-by: Moony <moony@hellomouse.net>
Co-authored-by: metalgearsloth <31366439+metalgearsloth@users.noreply.github.com>

lapatison committed:
Cig packs quick fix (#16152)

tom-leys committed:
Can open safe firelocks with a single click (#16104)

Slava0135 committed:
Hot potato on fire (#16017)

deltanedas committed:
make timers constructable and deconstructable (#15571)

Co-authored-by: deltanedas <@deltanedas:kde.org>

EmoGarbage404 committed:
anomaly commands (#16179)

ElectroJr committed:
Add more anomaly logs (#16178)

Vordenburg committed:
Resave every map (#15964)

Favored Nyano salvage maps.

Vordenburg committed:
Resave every map (#15964)

Favored Nyano salvage maps.

PuroSlavKing committed:
[Sprite] Diagonal walls (#15973)

ElectroJr committed:
Prevent projectiles from being affected by TryThrow() (#16185)

metalgearsloth committed:
Don't draw handitem offscreen (#16193)

TemporalOroboros committed:
Resolves SmesVisualizer is Obsolete (#13899)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

QuietlyWhisper committed:
cloth and health analyzer fit in med belt. (#16194)

Slava0135 committed:
Fix pinpointer flicker (#15452)

DrSmugleaf committed:
Fix db error when adding duplicate players in a round (#16206)

DrSmugleaf committed:
Fix admin logs duplicate id error when running tests (#16203)

TemporalOroboros committed:
Resolves ThrusterVisualizer is Obsolete (#13904)

Co-authored-by: metalgearsloth <comedian_vs_clown@hotmail.com>

Vordenburg committed:
Remove map saucer.

Vordenburg committed:
Let ReverseEngineering test ignore prototypes.

Necessary for now until we have a better solution to
reverse-engineering items that have recipes that result in empty
variants of themselves.

Vordenburg committed:
Add handcuff layers to some animals to pass tests.

These are just copied from the regular human.

Vordenburg committed:
Update usages of StaticPrice to DynamicPrice

Vordenburg committed:
Disable NPCSteering TimeOut for now

Vordenburg committed:
Wipe upstream changelog

Vordenburg committed:
Update Supply & Demand for mob and materialcomp

Vordenburg committed:
Fix arbitrage test

Vordenburg committed:
Update changelog

Vordenburg committed:
Merge pull request #1539 from Vordenburg/cherrypick-2023-04-18

Cherrypick 2023-04-18

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Nyanotrasen/Nyanotrasen into upstream-sync

crazybrain23 committed:
Fix cloth not going into lathes (#16231)

ElectroJr committed:
Fix client-side log spam (#16222)

eoineoineoin committed:
Fix material reclaimer (recycler) making sound on round start (#16230)

Co-authored-by: Eoin Mcloughlin <helloworld@eoinrul.es>

DEATHB4DEFEAT committed:
game kinda works wowowowow

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Park-station/Parkstation into upstream-sync

TemporalOroboros committed:
Resolves ApcVisualizer is Obsolete (#13898)

DEATHB4DEFEAT committed:
smonk

AJCM-git committed:
Fix buckling sounds (#16142)

metalgearsloth committed:
Gun + PKA fixes (#16244)

Vordenburg committed:
Merge pull request #1541 from Vordenburg/cherrypick-2023-05-08

Cherrypick 2023-05-08

Colin-Tel committed:
Update Angle and Shoukou (#1540)

Adds cell rechargers and material reclaimers, also changes the default setting to the brig timers to display what cells they correspond to.

FluffiestFloof committed:
Job coloring rework (#1504)

* Changes to id, job, encryption icons

Epi colors are now correct all around, fixes SalvTech not having an ID, Assistant icons are better, Valet icon is no longer a green A.

* Supervisors locale

* Spawner icons

Purely for mappers but I hope they like it

* Add mantis spawner icon

* Gladiator & MartialArtist job/id icons

* Service pass

* Cyborgs pass

* Service icon use same color as service ID

Slightly bothered me.

* A but slightly more spaced

* Reduce new sprites size

* Valet Icon

* Cadet similar A to assistant

* epistemic -> epistemics

Co-authored-by: Vordenburg <114301317+Vordenburg@users.noreply.github.com>

---------

Co-authored-by: Vordenburg <114301317+Vordenburg@users.noreply.github.com>

Vordenburg committed:
Add manual changelog tool (#1548)

Vordenburg committed:
Unify changelogs (#1547)

FluffiestFloof committed:
BoxEncryptionKey for Mysta (#1545)

Didn't rename them since there's another PR that does it

Elijahrane committed:
arachne rework (#1542)

* arachne rework

* remove legacy set, update oneirophages

IamVelcroboy committed:
Add ore to rock spawner (#1546)

Co-authored-by: Jeff <velcroboy333@hotmail.com>

Vordenburg committed:
Adjust events (#1549)

* Disable BureaucraticError

* Reduce intensity of SolarFlare

Vordenburg committed:
Adjust player min/max for Glacier & Lighthouse (#1550)

Vordenburg committed:
Update changelog (#1552)

jicksaw committed:
Fix Nyanotrasen Wiki update workflow (#1551)

* Update the Nyanotrasen Wiki API URL

* Run wiki updates only on branches with update-wiki environment

* Remove jsondump branch from wiki updates

I don't see any use for this yet. If we add it back, it should be a
protected branch.

Colin-Tel committed:
Hotfix angle.yml (#1553)

* Hotfix angle.yml

Changed Reporter door from Theatre access to Service access.

* Update angle.yml

* Update angle.yml

Fixed Epi's breakroom air vent

Elijahrane committed:
rename cyborgs to robots (#1543)

IamVelcroboy committed:
Map Edits (#1555)

* Map Edits

* fix grid atmos just to be safe

* Adjust emergency shuttle again

---------

Co-authored-by: Jeff <velcroboy333@hotmail.com>

FluffiestFloof committed:
Mothroaches (#1521)

* Mothroach init

* Removes the petting conflict

Conflicts with the picking up and it's just annoying honestly.

* Specific sound for it

* Fix incorrect .rsi

* Mothroach accent

* Refix the incorrect .rsi

Big brain moment

* Lowers density of the little bugger

* baseWalkSpeed, nyano tags.yml

* Mothroach 3kg

* Mothroach takes 3x more space as mouse

Just to match the 3x heavier than mouse

* Grindable, SmallMob, Moth damage modifier

Make the bugger grindable into blood and nutriment
Use the normal Moth damage modifier set, meaning they will have the same weakness/resistance
Lowers the asphyxiation rate to match mouse better

* Delete VentCritters.cs

* Update mothroachs to latest changes

Vordenburg committed:
Temporarily patch Discord changelog action (#1559)

Vordenburg committed:
Revert "Temporarily patch Discord changelog action (#1559)" (#1561)

This reverts commit 6eb490ac59772e33f9daef5e3cd8700720361e2d.

Vordenburg committed:
Fix robot spawners' job_id (#1563)

FluffiestFloof committed:
Custom borg healthanalyzer without battery (#1565)

ZeroDayDaemon committed:
Guidebook Tweaks (#1558)

* Guidebook Tweaks

* Update SL.xml

DebugOk committed:
Fix golem inventories (#1562)

IamVelcroboy committed:
Adds more suffixes (#1556)

* Adds more suffixes

* "Magic"

---------

Co-authored-by: Jeff <velcroboy333@hotmail.com>

Samsterious committed:
Adjust character name validation (#1566)

forkeyboards committed:
White Cane for Blind People (#1535)

* New White Cane

* Deleted unnecessary file

* The had a conflict in the PermanentBlindnessSystem.cs file so I pulled the new major 300+ changes update to my master and had Finket fix the .cs file I think

* Move BlindGear to Generic TraitGear in TraitSystem, Revert PermanentBlindnessComponent and System

TadJohnson00 committed:
Mk 32 Pistol, Mark 1 Rifle, Wakizashi, and more (#1472)

* Add Mk 32 Universal, Mk 1 Rifle, Katana Sheath

* Adds Mk 32 Universal - a robust .25 caseless pistol with enhanced offensive capabilities.

* Adds Mark 1 Rifle - a reliable and much-loved design for .30 light rifle. Also comes in tasteful ceremonial colours.

* Adds wakizashi - essentially a smaller katana.

* Adds katana belt - holds both a katana and a wakizashi.

* Adds CMO webbing - a chest rig in medical colours.

* Resprites katana to fit a more modern art style.

* Revises double barrel shotguns - they now automatically cycle.

* Added Mk32 Sprites

* Adds the sprites. I forgor

* Updated tags

* Added tag for MagazineUniversalCaselessRifle. Maybe this fixes it...

* Added belts to head lockers, changed uplink

* Added CMO rig to CMO locker

* Added katana sheath filled to HoS locker

* Changed the .30 rifle mag box to a .30 rifle speed loader box in uplink

* Reverted .25 caseless rubber magazine changes

* Reverted a change I made when tired. Will actually fix .25 caseless magazines and references properly later.

* Removed erroneous katana sheath sprites

* Removed erroneous sprites messing up the RSI validation.

* Removed Filled Katana Sheath from HoS locker

- Removed Filled Katana Sheath from HoS locker. Must now be mapped in manually.

* Fix typo in snipers.yml

Changed "anti-material" to "anti-materiel", as it should be

* Moved planned additions from upstream files

- Moved code occupying upstream files to Nyanotrasen folders
- Left code that directly overrides upstream files

* Fixed file swap issue

Files for caseless mags were swapped, now it should be rectified.

* Fixed another typo

Damn these things

* RSI file moved

Everything actually works now

* Updated belts.yml

Changed capacity of CMO webbing from 60 to 30

Fixed spacing between Martial Artist belt and Katana belt

* Minor fixes

- Edited a few mis-capitalisations
- Changed a couple of descriptions
- Removed erroneously added RSI

IamVelcroboy committed:
Adds Maintenance lights (#1554)

* Adds Maintenance lights

* Fix casing and add to spawners

* bump commit

* Forgot to save that one :)

* Remove disabler from item spawner

---------

Co-authored-by: Jeff <velcroboy333@hotmail.com>

IamVelcroboy committed:
Add suffix (#1567)

Co-authored-by: Jeff <velcroboy333@hotmail.com>

IamVelcroboy committed:
Fix/reenable salvage map atlas_medical (#1569)

Co-authored-by: Jeff <velcroboy333@hotmail.com>

DEATHB4DEFEAT committed:
kinda fix yaml validator

DEATHB4DEFEAT committed:
Merge branch 'master' into upstream-sync

DEATHB4DEFEAT committed:
migration (still broke)

Plyushsune committed:
turtle skirt (#1537)

* turtle skirt

this is my first real pr , be merciful

* yaml fixes

fixed some things in the yaml

* redos the sprites and places the yml in the correct spot

smh old me

* hot fix

* fixes yaml

idk

* fixes yaml again

farshdfjkghjugfgdd

* fixes yaml again

farshdfjkghjugfgdd

* Fixes yaml(again) and adds new variation

idk

* Relocates yaml to costumes and fixes sprites

idk

* adds outfits to vending machine

idk

* Final update

removes the basic one, renames the space one and modified the code names

Elijahrane committed:
cleanup turtle skirt commit (#1570)

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Nyanotrasen/Nyanotrasen into upstream-sync

DEATHB4DEFEAT committed:
map migration

DEATHB4DEFEAT committed:
oops

DEATHB4DEFEAT committed:
maps load

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Park-station/Parkstation into upstream-sync

DEATHB4DEFEAT committed:
deduplicate boxes rsi

DEATHB4DEFEAT committed:
resave our maps

DEATHB4DEFEAT committed:
fix invisible boxes

Colin-Tel committed:
Update Angle and Shoukou (#1574)

* Update Angle and Shoukou

Defibs! Maintenance lights! Wow!

* Update Angle and Shoukou

Added daishou, wakizashi, and other things, minor changes to Angle's medical, minor changes to Shoukou's atmos.

PPooch committed:
Defines earliestStart on MidRoundAntag at 25 (#1571)

Vordenburg committed:
Add new outfits by Hyenh (#1578)

* Fix female_none FemaleMask

* Add several new outfits by Hyenh

Samsterious committed:
Add Defibrillators and Cell Chargers to Medbays (#1575)

* Add defibrillators and cell chargers to Medbays

* Revert "Add defibrillators and cell chargers to Medbays"

This reverts commit 11ae123af8de5d4f56c30336b47d871bb08ea3cd.

* Add defibrillators and cell chargers to Medbays

* Revert changes to Arena, Hammurabi, and Tortuga

Vannn4 committed:
Add missing clothes to uniform printer (#1581)

* Adds missing uniforms, coats and mantles to uniform printer.

* Needed to rename Mystagogue's mantle

* Reverted change to .sln file

Elijahrane committed:
update changelog for past few days (#1583)

Elijahrane committed:
Fixes for #1581 (#1584)

* fix up uniform printer PR

* migrations

IamVelcroboy committed:
Map Edits (#1577)

* Map Edits

* Tortuga maint-light and armory update

---------

Co-authored-by: Jeff <velcroboy333@hotmail.com>

FluffiestFloof committed:
Fix robot crew monitor running out of charges (#1582)

* Rename to HandheldHealthAnalyzerBorg

* handheld crew monitor for robots

* Add no map just in case

Also some reorganization

* removes robot from analyser and monitor

The player doesn't really need to see that as they know they are a robot. At least I hope so.

* Modify phrasing of descriptions

* desc update

* Add more to the epinephrine hypospray desc

* Fixes

Elijahrane committed:
add comments to recent nyano changes to upstream Content.Server (#1579)

* add comments to recent nyano changed to Content.Server

* update format

* update format again

FluffiestFloof committed:
add/update comments to my previous PRs to be easily found (#1580)

* Borg and Mothroach marked

* Zombie

* obsolete in the near future

* Update format

IamVelcroboy committed:
Map Edits (#1590)

Co-authored-by: Jeff <velcroboy333@hotmail.com>

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Nyanotrasen/Nyanotrasen into upstream-sync

DEATHB4DEFEAT committed:
bigger character profile list preview

DEATHB4DEFEAT committed:
Merge branch 'master' of https://github.com/Park-station/Parkstation into upstream-sync

DEATHB4DEFEAT committed:
fix sprite views

Pspritechologist committed:
Merge branch 'master' of https://github.com/Park-Station/Parkstation into upstream-sync

DEATHB4DEFEAT committed:
Merge branch 'master' into upstream-sync

DEATHB4DEFEAT committed:
fix some announcements
